### PR TITLE
[WIP] iframe page

### PR DESCRIPTION
**Pull request by alcapat**
Add a page with links to test differents case of iframe settings

### CODE REVIEW

File: chsdi/templates/iframe.mako [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    param = request.params.get('param', defaultImageDisplay)
+%>    
+
+<!-- GESTION DES TOPICS -->
+% if param=='topic':
+
+<li> Topic ARE <br />
+<iframe src="${geoadmin_url}/embed.html?topic=are" name="Iframei_Topic_1"></iframe>
+<br />
+<br />
```
Comment (oterral) [inline chsdi/templates/iframe.mako]: Not good html.  You should have somthing like this:
```
<ul>
    <li>
        <span>Topic ARE<span><br/>
        <iframe src="${geoadmin_url}/embed.html?topic=are" name="Iframei_Topic_1"></iframe>
        <br/><br/>
     </li>
     <li> 
         <span>Topic Aviation</span>
         <iframe .....
     </li>
     .
     .
     .   
</ul>
```

**Comment by cedricmoullet** [comment]
@alcapat is it ready ?

**Comment by cedricmoullet** [comment]
@alcapat could you finalize it ?

**Comment by gjn** [comment]
@alcapat is this still relevant.

**Comment by gjn** [comment]
@alcapat Close or rebase?

**Pull request closed without merging**